docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
       - backend
 
   web:
+    ports:
+      - "127.0.0.1:8000:8000"
     environment:
```

Comment:
I moved this binding from `docker-compose.override.yml`, and added the fix for #761 at the same time.